document-generator-accounts/src/test/java/uk/gov/companieshouse/document/generator/accounts/AccountsDocumentInfoServiceImplTest.java

```diff
@@ -55,7 +60,7 @@ void testErrorThrownWhenFailedTransactionRetrieval() throws ServiceException {
         when(transactionService.getTransaction(anyString(), anyString())).thenThrow(new ServiceException("error"));
 
         assertThrows(DocumentInfoException.class, () ->
-                accountsDocumentInfoService.getDocumentInfo(createDocumentInfoRequest()));
+                accountsDocumentInfoService.getDocumentInfo(createDocumentInfoRequest(RESOURCE_URI_ABRIDGED)));
```

Comment:
could we format this file (and others) to meet our coding standards for line's lenght?

Reply:
we can, but all reformatting of the code should be done as a separate pr

Reply:
I thought the same after I posted it